url.bs

```diff
@@ -183,7 +183,11 @@ U+0020 SPACE, U+0022 ("), U+003C (&lt;), U+003E (&gt;), and U+0060 (`).
 <a>path percent-encode set</a> and U+002F (/), U+003A (:), U+003B (;), U+003D (=), U+0040 (@),
 U+005B ([) to U+005E (^), inclusive, and U+007C (|).
 
-<p>To <dfn>UTF-8 percent encode</dfn> a <a for=/>code point</a> <var>codePoint</var>, using a
+<p class=note>The <a><code>application/x-www-form-urlencoded</code></a> format's
+<a lt="urlencoded byte serializer">byte serializer</a> and the <a>URL parser</a>'s
+<a>query state</a> use <a>percent encode</a> directly without any of these sets.
+
+<p>To <dfn>UTF-8 percent encode</dfn> a <a for=/>code point</a> <var>codePoint</var> using a
```

Comment:
```suggestion
<p>To <dfn export>UTF-8 percent encode</dfn> a <a for=/>code point</a> <var>codePoint</var> using a
```

Reply:
Which spec will use this after https://github.com/whatwg/html/pull/5524?

Reply:
Ah, I mean, other spec (without s) 😅

AFAIK HTML doesn't depend on Shepherd but still `export` should indicate a term is potentially being used elsewhere, if anyone ever intend to remove it later.

Reply:
Ah no, not this specific one. My confusion.